documentation/modules/auxiliary/gather/advantech_webaccess_creds.md

```diff
@@ -0,0 +1,32 @@
+## Description
+
+This module allows you to log into Advantech WebAccess, and gather credentials from the user list.
```

Comment:
I **really** liked your previous description and how descriptive it was.  Maybe something like:
This module exploits three vulnerabilities in Advantech WebAccess.  The first vulnerability is the ability for an arbitrary user to access the admin user list page, revealing the username of every user on the system.  The second vulnerability is the user edit page can be accessed loaded by an arbitrary user, with the data of an arbitrary user.  The final vulnerability exploited is that the HTML Form on the user edit page contains the user's plaintext password in the masked password input box.  Typically the system should replace the actual password with a masked character such as "*".